modules/get-started/pages/licensing/monitor-license-status.adoc

```diff
@@ -0,0 +1,114 @@
++
+----
+warning: your license will expire soon
+----
+
+=== Change the output format
+
+Different output formats can be useful depending on your scenario. The available formats include JSON, YAML, text, and wide (expanded). For example, if you are writing scripts or automating license monitoring, you may prefer the JSON format, as it's easily parsed by tools like `jq` or integrated into monitoring systems.
+
+To get the license information in another format, use the `--format` flag:
+
+```bash
+rpk cluster license info --format <format>
+```
+
+Replace `<format>` with one of the available formats. See xref:reference:rpk/rpk-cluster/rpk-cluster-license-info.adoc[].
+
+For more details, including the SHA-256 checksum and Unix timestamp for expiration, use the `wide` format. This format is useful when you need a deeper level of detail about the license for troubleshooting or verification. For example:
+
+----
+LICENSE INFORMATION
+Organization:      Your Organization
+Type:              enterprise
+Expires:           Jan 1 2025
+License Expired:   false
+Checksum:          3d2e5b4d07d294534d9b7e90d8f73e91d34f7b4a62c45b6dc1c1e6f9d0a12bcf
+ExpiresUnix:       1735689600
+----
+
+[[admin]]
+== Check the status of a license with the Admin API
+
+You can retrieve the license status and information about in-use enterprise features using the Redpanda Admin API. This API provides JSON-formatted responses and is useful for programmatic access to the license details and feature status.
+
+To get the license status and features in use, make a `GET` request to the `/v1/features/enterprise` endpoint. For example:
+
+[,bash]
+----
+curl -X GET http://<redpanda-host>:<port>/v1/features/enterprise
+----
+
+Example response:
+
+[,json]
+----
+{
+  "license_status": "valid",
+  "violation": false,
+  "features": [
+    {
+      "name": "audit_logging",
+      "enabled": true
+    },
+    {
+      "name": "tiered_storage",
+      "enabled": false
+    }
+  ]
+}
+----
+
+- `license_status`: The status of the license. Possible values are:
+** `valid`: The license is valid.
+** `expired`: The license has expired.
+** `not_present`: No license is present.
+
+- `violation`: A boolean value that indicates whether there is a violation. For example, the license is not valid and one or more enterprise features are enabled.
+
+- `features`: An array of Enterprise features that lists each feature by name and whether it is enabled (`true`) or not (`false`).
+
+NOTE: The returned enterprise features are specific to Redpanda Enterprise Edition. They do not include xref:get-started:licensing/overview.adoc#connect[enterprise features for Redpanda Connect] or xref:get-started:licensing/overview.adoc#console[enterprise features in Redpanda Console].
+
+[[console]]
+== Check the status of a license in Redpanda Console
+
+You can also check the status of your license in the Redpanda Console UI. See xref:console:ui/add-license.adoc[].
+
+[[logs]]
+== Monitor a license in the logs
+
```

Comment:
Just wondered if there was an example of an enforcement action available/what sort of text a user might see?

Reply:
@pgellert do we have an example I can add to the doc?